crypto_kem/hqc-128-1-cca2/avx2_leaktime/gf2x.c

```diff
@@ -0,0 +1,144 @@
+/**
+ * \file gf2x.c
+ * \brief AVX2 implementation of multiplication of two polynomials
+ */
+
+#include "gf2x.h"
+#include "parameters.h"
+#include <stdint.h>
+#include <string.h>
+
+#define VECTORSIZE 256 /*!< The size in bits of a vector*/
+#define BYTESPERVECTOR (VECTORSIZE/8) /*!< The number of bytes in a vector*/
+#define BITSPERINT 32 /*!< The number of bits per int*/
+#define INTSPERVECTOR (VECTORSIZE/BITSPERINT) /*!< The number of ints per vector*/
+#define VEC_N_ARRAY_SIZE_VEC CEIL_DIVIDE(PARAM_N,256) /*!< The number of needed vectors to store PARAM_N bits*/
+typedef uint32_t v8uint32 __attribute__ ((vector_size (BYTESPERVECTOR), aligned (BYTESPERVECTOR)));/*!< v8uint32: a vector of 8 unsigned ints*/
```

Comment:
``__attribute__`` is a GNU compiler extension, and not available on Windows. You should use other tricks to enforce alignment – but this particular example should just use ``_m256i`` from ``immintrin.h``.

Reply:
I removed the avx2 implementation.